src/labels/CustomLabels.labels

```diff
@@ -2135,6 +2143,14 @@
         <shortDescription>Fixed Length Recurring Donation End Date must be valid</shortDescription>
         <value>End Date can&apos;t be in the future when Recurring Type is Fixed.</value>
     </labels>
+    <labels>
```

Comment:
Hard to believe we didn't have this as a label already.

Reply:
The best existing one I can find is "You must enable Enhanced Recurring Donations to use this component", the rest are more specific.

Reply:
There are a lot of super close Custom Label but none of them are exact